pystiche_papers/gatys_et_al_2017/utils.py

```diff
@@ -0,0 +1,30 @@
+import torch
+from torch import optim
+
+from pystiche.enc import MultiLayerEncoder, vgg19_multi_layer_encoder
+from pystiche.image.transforms import CaffePostprocessing, CaffePreprocessing
+
+__all__ = [
+    "gatys_et_al_2017_preprocessor",
+    "gatys_et_al_2017_postprocessor",
+    "gatys_et_al_2017_multi_layer_encoder",
+    "gatys_et_al_2017_optimizer",
+]
+
+
+def gatys_et_al_2017_preprocessor() -> CaffePreprocessing:
+    return CaffePreprocessing()
+
+
+def gatys_et_al_2017_postprocessor() -> CaffePostprocessing:
+    return CaffePostprocessing()
+
+
+def gatys_et_al_2017_multi_layer_encoder() -> MultiLayerEncoder:
```

Comment:
Make this more specific.
```suggestion
def gatys_et_al_2017_multi_layer_encoder() -> VGGMultiLayerEncoder:
```